composer.json

```diff
@@ -18,6 +18,8 @@
     ],
     "require": {
         "acquia/acquia-cms-starterkit": "^1.0",
+        "acquia/cohesion": "7.0.x-dev",
```

Comment:
@deepakmishra2 Can we please revert this ? As this shouldn’t be updated as part of this ticket.